src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -572,6 +576,10 @@ void ResumeCtrlImpl::SaveDataOnTimer() {
 
 void ResumeCtrlImpl::FinalPersistData() {
   LOG4CXX_AUTO_TRACE(logger_);
+  if (!resumption_storage_) {
+    LOG4CXX_ERROR(logger_, "resumption_storage_ is not initialized");
+    return;
+  }
```

Comment:
```suggestion
  DCHECK_OR_RETURN_VOID(resumption_storage_);
```
Agreed with @LuxoftAKutsan